app/views/admin/_sp_component.html.erb

```diff
@@ -0,0 +1,44 @@
+<%= link_to "Create new SP component", new_sp_component_path, class: "govuk-button"  %>
+
+<% components.reverse.each do |component| %>
+  <table class="govuk-table">
+    <caption class="govuk-table__caption" id="SpComponent/<%= component.id %>">SP component ID: <%= component.id %></caption>
+    <tbody class="govuk-table__body">
+      <tr class="govuk-table__row">
+        <th scope="row" class="govuk-table__header">Name</th>
+        <td class="govuk-table__cell"><%= component.name %></td>
+      </tr>
+      <tr class="govuk-table__row">
+        <th scope="row" class="govuk-table__header">VSP</th>
+        <td class="govuk-table__cell"><%= component.vsp %></td>
+      </tr>
+      <tr class="govuk-table__row">
+        <th scope="row" class="govuk-table__header">Team ID</th>
+        <td class="govuk-table__cell"><%= link_to(component.team_id, "/admin#teams/#{component.team_id}") if component.team_id %></td>
+      </tr>
+      <tr class="govuk-table__row">
+        <th scope="row" class="govuk-table__header">Environment</th>
+        <td class="govuk-table__cell"><%= component.environment %></td>
```

Comment:
`titleize`?

Reply:
Not sure whats better but I used `.capitalize` which capitalises the first character of the word, if you wanted to keep consistency when my PR gets merged ;)